src/cycax/cycad/cycad_part.py

```diff
@@ -163,18 +163,18 @@ def make_counterdrill(self, side: str, diameter: float, depth: float, angle: flo
         msg = "The adding of counterdrill to a side has not been implemented."
         raise NotImplementedError(msg)
 
-    def test_mesh_hole(self, x: float, y: float, z: float, diamter: float, depth: float):
+    def test_mesh_hole(self, x: float, y: float, z: float, diameter: float, depth: float):
```

Comment:
Method starting with `test_` confuses the reader, one wonders if this method is called by PyTest. Seems not to be used anywhere so you might want top remove it.